src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -254,6 +254,16 @@ public static IServiceCollection AddMvc([NotNull] this IServiceCollection servic
             return WithControllersAsServices(services, controllerTypes.Select(type => type.AsType()));
         }
 
+        public static IServiceCollection AddMvcLocalization([NotNull] this IServiceCollection services)
+        {
+            services.ConfigureRazorViewEngine(options =>
+            {
+                options.ViewLocationExpanders.Insert(0, new LanguageViewLocationExpander());
```

Comment:
Should this just `Add` \ `Append` instead of being first? This way if the user adds a expander, and calls this, their ordering is preserved.